requirements.txt

```diff
@@ -1,17 +1,5 @@
-beautifulsoup4==4.6.3
-certifi==2022.12.7
-chardet==3.0.4
-charset-normalizer==3.1.0
-click==8.1.3
-Flask==1.0.2
-gunicorn==19.9.0
-idna==2.7
-itsdangerous==2.1.2
-Jinja2==3.1.2
-lxml==4.9.2
-MarkupSafe==2.1.2
-requests==2.20.0
-robobrowser==0.5.3
-six==1.16.0
-urllib3==1.24.3
-Werkzeug==2.2.3
+beautifulsoup4
```

Comment:
It will be better to add some version constraints, considering python's immaculate backwards compatibility.

Reply:
Due to the same immaculate backwards compatibility, adding versions would break this in the future. So it doesn't really matter. If the libraries don't break their API, this should be the better option imo.